### DISCUSSION TITLE
Sunsynk / HA Help

### DISCUSSION THREAD

**Discussion by MatimbaSolutions** [kellerza/sunsynk#83] [Q&A] [answered]
Hi, please give me some advice, can this HA addon work via my HA and normal WIFI via the Wifi dongle to my Inverter, or do i need a RJ45 cable and adaptor pluged into my Raspberry PI

**Comment by duanemck** [comment, marked as answer]
It needs a physical cable connection to the inverter. There is a 3rd party WiFi dongle you can get called a Solarman and a HA plugin for that if you want a wireless connection. But the default WiFi dingle only connects to the Sunsynk servers for their app.

**Comment by MatimbaSolutions** [comment]
Thanks for the reply, I can only have the Sunsync or the Solarman dongle?
not both? Then would you recommend to get this
https://www.robotics.org.za/index.php?route=product/product&product_id=5947
Converter?

**Reply by duanemck** [reply]
I believe so, I think they both plug into the single RS232 port.

I haven't used that one and I'm not 100% where it would connect on the inverter.

I bought this one as the inverter had a RJ45 jack for RS485:

https://solar-assistant.io/shop/products/sunsynk_rs485 (make sure you select the ZAR option in the drop down, it default to USD)

**Reply by duanemck** [reply]
Your link would probably work but you would have to crimp the RJ45 plug on yourself.

**Reply by jabezz** [reply]
The one you linked on robotics.org.za works perfectly fine. I'm using it.
Just follow the instructions on the readme https://github.com/kellerza/sunsynk
And yes, you just crimp the RJ45 yourself (pretty straight forward from the instructions)

**Comment by MatimbaSolutions** [comment]
Thanks, I will order one now, i have another dongle here currently, but not
working very good

**Comment by kellerza** [comment]
I use the Wave one also from Micro Robotics, also very nice with ESD protection. As long as you have a GND in addition to the data pins it should be fine!
